openshift/template.yaml

```diff
@@ -168,7 +168,7 @@ objects:
                 name: bayesian-config
                 key: auth-url
           - name: SQS_MSG_LIFETIME
-            value: 24
+            value: "24"
```

Comment:
This should be template parameterised. Look at the parameter section & do accordingly.